apps/testingaccessibility/src/server/get-purchased-product.ts

```diff
@@ -0,0 +1,55 @@
+import {getToken} from 'next-auth/jwt'
+import {getSdk} from '../lib/prisma-api'
+import get from 'lodash/get'
+import last from 'lodash/last'
+import {sanityClient} from '../utils/sanity-client'
+import groq from 'groq'
+
+const defaultProductQuery = groq`*[_type == "product" && productId == $productId][0]{
+  productId,
+  modules[]->{
+    "slug": slug.current,
+    title,
+    sections[]->{
+      "slug": slug.current,
+      title,
+      lessons[]->{
+        "slug": slug.current,
+        title
+      }
+    }
+  }
+  }`
```

Comment:
this looked like it was the biggest and most complete so I used it as a default

Reply:
looks right to me and everything seems to be working 👍